server/pxf-service/src/templates/tomcat/bin/setenv.sh

```diff
@@ -24,8 +25,22 @@ JAVA_AGENTS=""
 JAVA_LIBRARY_PATH=""
 
 # DO NOT EDIT VALUES FOR THE VARIABLES BELOW -- they are generated by the start script
-PXF_OPTS="-Dconnector.https.port=8443 -Dpxf.home=${PXF_HOME} -Dpxf.conf=${PXF_CONF} -Dconnector.http.port=${PXF_PORT} -Dbase.shutdown.port=${PXF_SHUTDOWN_PORT} -Dpxf.log.dir=${PXF_LOGDIR} -Dpxf.service.user.impersonation.enabled=${PXF_USER_IMPERSONATION} -Dpxf.service.fragmenter.cache.enabled=${PXF_FRAGMENTER_CACHE} -Dpxf.service.kerberos.keytab=${PXF_KEYTAB} -Dpxf.service.kerberos.principal=${PXF_PRINCIPAL}"
-JAVA_OPTS="${PXF_JVM_OPTS} $AGENT_PATHS $JAVA_AGENTS $JAVA_LIBRARY_PATH $PXF_OPTS"
+PXF_OPTS="-Dconnector.https.port=8443 -Dpxf.home=$PXF_HOME -Dpxf.conf=$PXF_CONF -Dconnector.http.port=$PXF_PORT -Dbase.shutdown.port=$PXF_SHUTDOWN_PORT -Dpxf.log.dir=$PXF_LOGDIR -Dpxf.service.user.impersonation.enabled=$PXF_USER_IMPERSONATION -Dpxf.service.fragmenter.cache.enabled=$PXF_FRAGMENTER_CACHE -Dpxf.service.kerberos.keytab=$PXF_KEYTAB -Dpxf.service.kerberos.principal=$PXF_PRINCIPAL"
+if [ "$PXF_OOM_KILL" = true ]; then
+  PXF_OPTS="$PXF_OPTS -XX:OnOutOfMemoryError='${PXF_HOME}/pxf-service/bin/kill-pxf.sh %p'"
+fi
+if [ -n "$PXF_OOM_DUMP_PATH" ]; then
+  cat <<EOF
+=========================================================================================
+  NOTICE: Setting Tomcat options HeapDumpOnOutOfMemoryError and HeapDumpPath
```

Comment:
@frankgh Please review this NOTICE to the user.

Reply:
These are not tomcat options.
```suggestion
  NOTICE: Adding JVM options -XX:+HeapDumpOnOutOfMemoryError and -XX:+HeapDumpPath
```